x-pack/plugins/task_manager/README.md

```diff
@@ -55,6 +55,7 @@ The task_manager can be configured via `taskManager` config options (e.g. `taskM
 - `monitored_stats_running_average_window`- Dictates the size of the window used to calculate the running average of various "Hot" stats. Learn More: [./MONITORING](./MONITORING.MD)
 - `monitored_stats_required_freshness` - Dictates the _required freshness_ of critical "Hot" stats. Learn More: [./MONITORING](./MONITORING.MD)
 - `monitored_task_execution_thresholds`- Dictates the threshold of failed task executions. Learn More: [./MONITORING](./MONITORING.MD)
+- `internal.exclude_task_types` - Which task types to exclude from execution by ensuring they are never claimed. Supports wildcard usage (such as `actions:*`). This is intended for debugging use only
```

Comment:
```suggestion
- `internal.exclude_task_types` - Which task types to exclude from execution by ensuring they are never claimed. Supports wildcard usage (such as `actions:*`). This is an experimental config and should be used with caution.
```

Reply:
Yea this is interesting. I don't know how to phrase this exactly. cc @mikecote for any advice

Reply:
Maybe something like this? cc @gchaps for opinions.

```suggestion
- `internal.exclude_task_types` - A list of task types to exclude from running. Supports wildcard usage (such as `namespace:*`). This configuration is experimental, unsupported and can only be used for temporary debugging purposes as it will cause Kibana to behave in unexpected ways.
```

Reply:
Best of both worlds! Thanks all!